### PR TITLE
Initial github actions setup

### PR DESCRIPTION
**Pull request by hikhvar**

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Go Build
```
Comment (arnisoph) [inline .github/workflows/go.yml]: why not use the Makefile here?

Comment (hikhvar) [reply .github/workflows/go.yml]: I'm not sure if Make is installed in the go action. This is the Go action template provided by Github.

Comment (hikhvar) [reply .github/workflows/go.yml]: Switched to make

**Comment by arnisoph** [comment]
thx 👋

**Pull request merged**